.gitignore

```diff
@@ -85,3 +85,4 @@ ModulesMaintainers.txt
 Nits.txt
 Pylint.txt
 YAMLLint.txt
+myresults.xml
```

Comment:
Should be removed

Reply:
These files are created if you try to run babblesim tests from local directory. It is annoyance to ignore them everytime.